### PR TITLE
Standardize repo

### LINKED ISSUE #22: Bring repo up to standards and release
- [x] Update `conf.py` (@stevepiercy)
- [x] Update `contributing.md` (@stevepiercy)
- [x] Add `RELEASING.txt` (@stevepiercy)
- [x] Update `README.rst` (@stevepiercy)
- [x] `setup.py`: bump Sphinx version, update trove classifiers
- [x] `tox.ini`: drop 3.3, add 3.5, 3.6, 3.7 nightly as allowed failure, use matrix(?)
- [x] `travis.yml`: drop 3.3, add 3.5, 3.6, 3.7 nightly as allowed failure
- [x] get [complete test coverage on #20](https://travis-ci.org/Pylons/pyramid_chameleon/jobs/386169168#L554)
- [ ] other?

### PR DESCRIPTION
**Pull request by stevepiercy**
- fix #22

### CODE REVIEW

**Comment by stevepiercy** [comment]
I think this is as good as I can get it. There is some [missing coverage](https://travis-ci.org/Pylons/pyramid_chameleon/jobs/386169199#L562) on the recent PR #20. Otherwise I think I got it all configured.

**Comment by stevepiercy** [comment]
Ping @bertjwregeer @tseaver for review. Is this OK to merge? My changes are not the cause of the failing test covereage.

File: tox.ini [hunk 1]
```diff
@@ -1,31 +1,64 @@
...
 # cobertura jenkins reporting and b) pypy and jython can't handle any
 # combination of versions of coverage and nosexcover that i can find.
 
+[testenv:coverage]
+basepython = python3.5
+commands =
+    coverage erase
+    coverage combine
+    coverage xml -i
+    coverage report --show-missing --fail-under=100
```
Comment (bertjwregeer) [inline tox.ini]: Remove `--fail-under=100` until we get a PR that fixes things and gets tests back to 100% coverage.

**Comment by stevepiercy** [comment]
Do not merge yet. I also want to move 3.7 into a different build in travis, a la Pyramid.

File: tox.ini (hunk 1 above)
Comment (stevepiercy) [reply tox.ini]: Fixed in 9ff06f2

**Comment by stevepiercy** [comment]
@bertjwregeer @tseaver Builds pass, although coverage from the merged #20 still needs some work. Ready for review again.

File: setup.py [hunk 2]
```diff
@@ -38,8 +38,9 @@
         "Programming Language :: Python :: 2",
         "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.3",
         "Programming Language :: Python :: 3.4",
+        "Programming Language :: Python :: 3.5",
+        "Programming Language :: Python :: 3.6",
```
Comment (tseaver) [inline setup.py]: If we are testing on 3.7, then we should add it here.

Comment (stevepiercy) [reply setup.py]: Fixed in e73e8ba

**Comment by stevepiercy** [comment]
@tseaver per your comment, I added Python 3.8 to setup.py, too. Please let me know if I should revert that. Otherwise is this PR OK to merge?

File: CHANGES.txt [hunk 3]
```diff
@@ -1,6 +1,9 @@
 0.4 (unreleased)
 ----------------
 
+- Update the repo to standards.  See
+  https://github.com/Pylons/pyramid_chameleon/issues/22
+
```
Comment (tseaver) [inline CHANGES.txt]: This may need merging, after today's commit dropping 3.3 support and adding 3.4, 3.5, 3.6, and 3.7 support.

Comment (stevepiercy) [reply CHANGES.txt]: Rebased on master, merged differences, pushed, build succeeds. @tseaver could I get one more review from you, please? Thank you!

Comment (tseaver) [reply CHANGES.txt]: I take it you mean to bump the next release to 1.0 in a separate PR after merging this one?

Comment (stevepiercy) [reply CHANGES.txt]: Yup. 1.0 sounds good to me!

**Pull request merged**